openmetadata-ui/src/main/resources/ui/src/axiosAPIs/miscAPI.ts

```diff
@@ -275,3 +276,7 @@ export const getEntityCount = async (
 
   return response.data;
 };
+
+export const getAllEntityCount = (): Promise<AxiosResponse<EntitiesCount>> => {
+  return APIClient.get('/util/entities/count');
+};
```

Comment:
It would be great if we follow the async/await pattern here and send a response from here instead of sending the full Axios object!
Let me know if you need any help related to it!

Reply:
Done